.github/workflows/ci-actions-incremental.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches-ignore:
       - 'dependabot/**'
-      - 'jakarta-rewrite'
```

Comment:
@famod could you have a look at the CI changes below.
I have to exclude a few modules and I thought transforming the GIB output was the best option.

My understanding is that, in the case of a push to the `jakarta-rewrite` branch, it won't be a PR and it will compare to `main`, which is what I want.

Reply:
LGTM, but two remarks/questions:
- changes made by `./jakarta/prepare.sh` will not contribute to the change detection due to `-Dgib.uncommitted=false`...is this what you want?
- I think you could alternatively use `-pl !...` to exclude those modules but the `grep` looks ok and I suppose it's only a temporary thing

Reply:
`./jakarta/prepare.sh` is just about pushing some snapshots to the Maven repo, it doesn't change the code. The branch contains all the code changes.
Problem I saw with `-pl` was what happens if I exclude all of them? Also it was a bit less convenient as I needed to do it for the JDK tests and the native tests. And for the native tests, I would have had to only exclude those that are actually running as the build is split.

And yes, it's just temporary, we will drop this once we move to Jakarta. Well and before if we can get these modules to pass.